sbncode/EventGenerator/MeVPrtl/Tools/TxtFileGen_tool.cc

```diff
@@ -251,19 +244,20 @@ simb::MCFlux TxtFileGen::GetNext() {
 }
   
 simb::MCFlux TxtFileGen::MakeMCFlux(std::string line) {
-  std::cout << "Parsing line: " << line << std::endl;
+  // std::cout << "Parsing line: " << line << std::endl;
 
   // read the 4-vector off the line
-  double E, px, py, pz;
+  double E, px, py, pz, npot, wgt;
+  int pdgcode;
   std::stringstream l4vec(line);
-  l4vec >> px >> py >> pz >> E;
+  l4vec >> px >> py >> pz >> E >> pdgcode >> wgt >> npot;
 
-  std::cout << "Values: " << px << " " << py << " " << pz << " " << E << std::endl;
+  std::cout << "Values: " << px << " " << py << " " << pz << " " << E << " " << pdgcode << " " << wgt << std::endl;
```

Comment:
I see in some other cases we are getting rid of the cout. 
Shall we do the same here?

Reply:
Fixed.